src/main/java/org/opensearch/repository/encrypted/security/CryptoIO.java

```diff
@@ -17,6 +18,8 @@
 
 public class CryptoIO implements Encryptor, Decryptor {
 
+    public static final int BUFFER_SIZE = (1 << 17) + 1; // 128KB default mark/reset buffer size
```

Comment:
Just curious, why not  `128 * 1024` or even ByteSizeUnit`? (no need to comment what that means)